Candidate: CVE-2019-16228
PublicDate: 2019-09-11 15:15:00 UTC
References:
 https://github.com/TeamSeri0us/pocs/tree/master/lmdb/FPE
 https://www.cve.org/CVERecord?id=CVE-2019-16228
Description:
 An issue was discovered in py-lmdb 0.97. There is a divide-by-zero error in
 the function mdb_env_open2 if mdb_env_read_header obtains a zero value for
 a certain size field. NOTE: this outcome occurs when accessing a data.mdb
 file supplied by an attacker.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]


Patches_py-lmdb:
upstream_py-lmdb: needs-triage
precise/esm_py-lmdb: DNE
trusty_py-lmdb: ignored (end of standard support)
trusty/esm_py-lmdb: DNE
esm-infra-legacy/trusty_py-lmdb: DNE
xenial_py-lmdb: ignored (end of standard support, was needs-triage)
esm-apps/xenial_py-lmdb: needs-triage
bionic_py-lmdb: ignored (end of standard support, was needs-triage)
esm-apps/bionic_py-lmdb: needs-triage
disco_py-lmdb: ignored (end of life)
eoan_py-lmdb: ignored (end of life)
focal_py-lmdb: needs-triage
esm-apps/focal_py-lmdb: needs-triage
groovy_py-lmdb: ignored (end of life)
hirsute_py-lmdb: ignored (end of life)
impish_py-lmdb: ignored (end of life)
jammy_py-lmdb: needs-triage
esm-apps/jammy_py-lmdb: needs-triage
kinetic_py-lmdb: ignored (end of life, was needs-triage)
lunar_py-lmdb: ignored (end of life, was needs-triage)
mantic_py-lmdb: ignored (end of life, was needs-triage)
noble_py-lmdb: needs-triage
esm-apps/noble_py-lmdb: needs-triage
oracular_py-lmdb: needs-triage
plucky_py-lmdb: needs-triage
devel_py-lmdb: needs-triage
